test/unit/cli_tests.rb

```diff
@@ -736,6 +774,63 @@ class ReleaseCommandTests < IOCommandTests
 
   end
 
+  class CommandSetTests < UnitTests
```

Comment:
@jcredding here is me adding a bunch of command set unit tests like you suggested - this turned out to be really nice actually so thanks for suggesting!